demisto_sdk/commands/init/initiator.py

```diff
@@ -720,45 +912,100 @@ def script_init(self) -> bool:
 
         return True
 
-    def yml_reformatting(self, current_suffix: str, integration: bool = False):
-        """Formats the given yml to fit the newly created integration/script
+    def modeling_or_parsing_rules_yml_reformatting(self, current_suffix: str):
```

Comment:
Fix the description of this function
Also all files in modeling rules/parsing rules missing the correct suffix